### PR TITLE
Add initial contributing documentation

### PR DESCRIPTION
**Pull request by Siilwyn**

### CODE REVIEW

File: contributing.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+The quickest and most open way to **communicate with the Ayria team** is on our [**Discord channel**](https://discord.gg/Jg8NZDK).
+
+### GitHub Issues
+A bit more formal way of communication with fellow Ayria members, but a little less quick and convenient like the chat. Submit an issue when you run into problems or just would like to discuss a certain topic, be it _features, code style, code inconsistencies, minor changes and fixes, etc._
+
+### Pull Requests
+It’s fine to just submit a small pull request without first making an issue or asking in the chat unless if it’s a significant change that will require **planning and reviewing**. Also see [Creating a Pull Request](#creating-a-pr) and [Git Style Guidelines](#git-style-guidelines).
+
+## Possible Contributions
+
```
Comment (Convery) [inline contributing.md:16]: * Pipe IO for fetching account information etc..

Comment (Convery) [reply contributing.md:16]: * Callback for whitelisting manually installed plugins [Bootstrap_cpp ref](https://github.com/AyriaPublic/Bootstrapmodule_cpp/blob/master/Source/Bootstrapper/Pluginloader.cpp#L42-L48)

Comment (Siilwyn) [reply contributing.md:16]: That's not something that can be worked on right now, right? Same for package management.

Comment (Convery) [reply contributing.md:16]: I mean, a system for opening a pipe and reading/writing data can be done at any time. Fetching the actual data requested however.. =P

**Pull request merged**